internal/network/command.go

```diff
@@ -43,6 +44,11 @@ type command struct {
 	*pcmd.AuthenticatedCLICommand
 }
 
+var (
+	ConnectionTypes = []string{"privatelink", "peering", "transitgateway"}
```

Comment:
looks like this var is only used in this file. What's the reason for exposing them?

Reply:
We will be using it for other commands as well. For example, once the `MultipleSearchFilter` bug is fixed by the API team, we will use it for the `network list` command.

Reply:
👍